### PR TITLE
ack: clarify file types syntax meaning

### LINKED ISSUE #2805: ack: meaningless value
In [ack](https://github.com/tldr-pages/tldr/blob/master/pages/common/ack.md): `ack --ruby {{each_with_object}}`

What does it mean "each_with_object"? I tried to find out from [ack documentation](https://beyondgrep.com/documentation/), but without success.
Please use a more descriptive value name!

### PR DESCRIPTION
**Pull request by negamorgan**
fix #2805

- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#guidelines).

### CODE REVIEW

File: pages/common/ack.md [hunk 1]
```diff
@@ -6,9 +6,9 @@
 
 `ack {{foo}}`
 
-- Find files in a specific language:
+- Find files scoped to a specific programming language:
```
Comment (agnivade) [inline pages/common/ack.md]: Pardon my ignorance, but what does "scoped" exactly mean here ? Will it only check files with ".rb" extension ? Or does it mean something else ?

Comment (negamorgan) [reply pages/common/ack.md]: Pretty much. From the docs: 

> The following is the list of filetypes supported by ack.  You can specify a file type with the --type=TYPE format, or the --TYPE format.  For example, both --type=perl and --perl work.

> Note that some extensions may appear in multiple types.  For example, .pod files are both Perl and Parrot.

> --[no]ruby         .rb .rhtml .rjs .rxml .erb .rake .spec; Rakefile; first line matches /^#!.*\bruby/

Comment (agnivade) [reply pages/common/ack.md]: Then I think the previous description was clearer. By "scope", I was thinking of "variable scoping" and was wondering how does this apply to files.

Comment (agnivade) [reply pages/common/ack.md]: Made the changes myself.

**Pull request merged**